torch/csrc/jit/passes/quantization.cpp

```diff
@@ -601,12 +568,7 @@ void InsertQuantDeQuantImpl(
             InsertQuantDeQuantImpl(m.value(), module_method_name);
           }
         }
-        if (v->node()->kind() == prim::GetAttr &&
-            v->node()->s(c10::attr::name) == "bias") {
-          qh.quantizeBias(v);
-        } else {
-          qh.quantizeTensor(v);
-        }
+        qh.quantizeTensor(v);
```

Comment:
It seems that after this change we will be quantizing bias - we would just do that in the same manner as a usual tensor. Is it intended or do we want to skip bias tensors completely?

Reply:
oh, I'll restore the check

Reply:
to support this properly we'll probably need a value set as well.. but we can do this later